extra/release_tool.py

```diff
@@ -2409,6 +2414,42 @@ def do_build(args):
     trigger_build(state, tag_avail)
 
 
+def determine_version_bump(state, repo, from_v, to_v):
+    revlist = execute_git(
+        state, repo.git(), ["rev-list", "%s..%s" % (from_v, to_v)], capture=True
+    ).split("\n")
+
+    (major, minor, patch, _) = version_components(from_v)
+    version_mask = [False, False, False]
+
+    git_cmd = ["log", "--format=%B", "-1", sha]
+    for sha in revlist:
+        commit_message = execute_git(state, repo.git(), git_cmd, capture=True)
+        m = re.search(r"^BREAKING CHANGE:.+", commit_message, re.MULTILINE)
```

Comment:
Here it is necessary cuz re.MULTILINE will make caret^ match on each newline, not just the start of the string. "BREAKING CHANGE" can not appear on the first line in a commit msg, since that wouldt be a conventional commit. 

However, its not necessary on the other regex match function call, as we are trying to match on the start of the string; not after any newline.

Reply:
You are absolutely right of course! 

Good work :medal_military: